.github/workflows/repo.yaml

```diff
@@ -63,7 +61,6 @@ jobs:
       - uses: actions/checkout@v4
       - uses: pnpm/action-setup@v4
         with:
-          version: 8
```

Comment:
Is the idea that this isn't needed any longer since the action pays attention to `packageManager` within the package.json?

Reply:
Yep, it complains if it's present in both places